dali/kernels/slice/slice_kernel_test.h

```diff
@@ -94,26 +94,48 @@ class SliceTest : public ::testing::Test {
       const auto *in_tensor = in.tensor_data(i);
       auto *out_tensor = out.tensor_data(i);
 
+      int channel_dim = slice_args[i].channel_dim;
+      auto *fill_values = slice_args[i].fill_values.data();
+      int fill_values_size = slice_args[i].fill_values.size();
+      if (fill_values_size > 1) {
+        ASSERT_NE(-1, channel_dim);
```

Comment:
```suggestion
        ASSERT_GT(0, channel_dim);
```

Reply:
channel_dim 0 is also valid. I can use ASSERT_GE